client/app/student/dashboard/dashboard.controller.js

```diff
@@ -10,8 +10,12 @@ angular.module('venueApp')
     Auth.getCurrentUser((user) => {
       $scope.user = user;
 
-      User.get({id: user._id, withCourses:true, withEvents: true, withEventSections: true}, (usr) => {
-        $scope.user = usr;
+      User.get({id: user._id, withCourses:true, withEvents: true, withEventSections: true}, (user) => {
+        $scope.user = user;
+        $scope.coursesLength = angular.equals(user.courses,{})?0:1
+
+        $scope.eventsLength = angular.equals(user.events, {})?0:1
```

Comment:
It check if there is any. It evolved from the previous method when we had arrays (unfortunately its not efficient to directly check the number of keys in objects), and I forgot to change the variable name. I can change it